v3/src/styles/menu-style.ts

```diff
@@ -6,7 +6,7 @@ const helpers = createMultiStyleConfigHelpers(["menu", "list", "item"])
 export const Menu = helpers.defineMultiStyleConfig({
   baseStyle: {
     menu: {
-      font: `"Montserrat-Regular", sans-serif`,
+      font: `"Montserrat", sans-serif`,
```

Comment:
Super nitpicky, sorry, but it'd be a little nicer if this was changed to...

`font: "Montserrat, sans-serif",`

Similarly, maybe for consistency remove all the `"` and `'` instances around `Montserrat` in the SCSS files, too, since they're no longer needed?

None of that's required, of course, if you don't think it's worth bothering over. The fonts are all being applied appropriately as is.

Reply:
Good point -- done.